panel/tests/ui/widgets/test_sliders.py

```diff
@@ -0,0 +1,358 @@
+import time
+
+import pytest
+
+try:
+    import playwright  # noqa
+except ImportError:
+    pytestmark = pytest.mark.skip('playwright not available')
+
+pytestmark = pytest.mark.ui
+
+from panel.io.server import serve
+from panel.widgets import (
+    EditableFloatSlider, EditableIntSlider, EditableRangeSlider,
+)
+
+
+class _editable_text_input:
+    """
+    This function is added for convenience, as it is pretty
+    cumbersome to find and update the value of Editable text input.
+    """
+
+    def __init__(self, page, nth=0):
+        self.page = page
+        self.text_input = page.locator("[placeholder=\"\\30 \"]").nth(nth)
+
+    @property
+    def value(self):
+        # Needs reload of page to update attribute
+        # Should be reimplemented without the need for update
+        self.page.reload()
+        return self.text_input.get_attribute("value")
+
+    @value.setter
+    def value(self, value):
+        self.text_input.fill(str(value))
+        self.text_input.press("Enter")
+
+
+@pytest.mark.parametrize(
+    "widget,val1,val2,val3,func",
+    [
+       (EditableIntSlider, 25, 24, 20, int),
+       (EditableFloatSlider, 25.5, 24.5, 19.5, float),
+    ],
+    ids=["EditableIntSlider", "EditableFloatSlider"]
+)
+def test_editableslider_textinput_end(page, port, widget, val1, val2, val3, func):
+    widget = widget()
+
+    serve(widget, port=port, threaded=True, show=False)
+    time.sleep(0.2)
+
+    page.goto(f"http://localhost:{port}")
+
+    text_input = _editable_text_input(page)
+
+    text_input.value = val1
+    page.wait_for_timeout(200)
+    assert widget.value == val1
```

Comment:
The `wait_until` utility wasn't working?

Reply:
A classic case of `I forgot it`...